src/app/page/MissionDetails/MissionDetails.jsx

```diff
@@ -51,7 +51,7 @@ const MissionDetailsPage = ({ history, match }) => {
       setMission(mission);
     };
     fetchMissionById();
-  }, []);
+  }, [match.params.id]);
```

Comment:
https://github.com/facebook/react/issues/14920

Wow. Eslint react exhaustive-deps rule checks for issues with useEffect. Dope

Reply:
I just proposed to add this to my work project :P